packages/ember-debug/lib/deprecate.js

```diff
@@ -95,8 +95,9 @@ export let missingOptionsUntilDeprecation = 'When calling `Ember.deprecate` you
 
 /**
   Display a deprecation warning with the provided message and a stack trace
-  (Chrome and Firefox only). Ember build tools will remove any calls to
-  `Ember.deprecate()` when doing a production build.
+  (Chrome and Firefox only). Ember build tools will remove any calls
+  to `Ember.deprecate()` when doing an Ember.js framework production build
+  and will make the assertion a no-op for an application production build.
```

Comment:
Seems like it should say `deprecation`. Is that really the app-space behavior for deprecate in prod?